.drone.yml

```diff
@@ -29,7 +29,8 @@ steps:
     path: /var/run
   commands:
   - export VMCK_URL=http://$VMCK_IP:$VMCK_PORT
-  - docker run -e VMCK_URL=$VMCK_URL vagrant-vmck bash /src/run-test.sh
+  - export JOB_NAME="drone $DRONE_REPO:$DRONE_BRANCH"
```

Comment:
Could we also include the [build number](https://docker-runner.docs.drone.io/configuration/environment/variables/drone-build-number/) please?